aiida/tools/visualization/graph.py

```diff
@@ -686,7 +701,11 @@ def recurse_descendants(
 
         # Add all traversed nodes to the graph with default styling
         for _, traversed_node in traversed_nodes.items():
-            self.add_node(traversed_node, style_override={})
+            if target_cls and not isinstance(traversed_node, target_cls):
+                style_override = self._ignre_node_override_style
+                self.add_node(traversed_node, style_override=style_override)
+            else:
+                self.add_node(traversed_node, style_override={})
```

Comment:
(1) So, maybe I would just call this `_ignored_node_style`? (writing "ignored" fully for clarity and removing "override" since this is just a style, the action of overriding happens when you pass it to `add_node` in the corresponding parameter).

(2) Also, notice that the default for `style_override` is `None`. Although currently passing the empty dict has the same effect, it is conceivable (though I admit a bit unlikely) that this could be re-purposed in the future. Just for consistency and to be on the safe side I would advise to use `None`.

Reply:
Sorry for the typo here :sweat_smile: .  Changed as your advise.